### PR TITLE
Replace $verbose with VERBOSITY_VERBOSE for scanFiles method

### PR DESCRIPTION
**Pull request by kesselb**
Follow-up https://github.com/nextcloud/server/pull/11831

This pr replace `if ($verbose)` with `OutputInterface::VERBOSITY_VERBOSE`

### CODE REVIEW

File: apps/files/lib/Command/ScanAppData.php [hunk 1]
```diff
@@ -96,44 +83,36 @@ protected function scanFiles($verbose, OutputInterface $output) {
...
-				$this->abortIfInterrupted();
-			});
-			$scanner->listen('\OC\Files\Utils\Scanner', 'scanFolder', function () use ($output) {
-				$this->foldersCounter += 1;
-				$this->abortIfInterrupted();
-			});
-		}
-		$scanner->listen('\OC\Files\Utils\Scanner', 'scanFile', function($path) use ($output) {
+		$scanner->listen('\OC\Files\Utils\Scanner', 'scanFile', function ($path) use ($output) {
+			$output->writeln("\tFile\t<info>$path</info>", OutputInterface::VERBOSITY_VERBOSE);
```
Comment (ChristophWurst) [inline apps/files/lib/Command/ScanAppData.php]: please don't change these unrelated commands. It's fine to *fix*/*enhance* this in another PR though.

File: apps/files/lib/Command/ScanAppData.php [hunk 2]
```diff
@@ -96,44 +83,36 @@ protected function scanFiles($verbose, OutputInterface $output) {
...
+			++$this->filesCounter;
+			$this->abortIfInterrupted();
+		});
+
+		$scanner->listen('\OC\Files\Utils\Scanner', 'scanFolder', function ($path) use ($output) {
+			$output->writeln("\tFolder\t<info>$path</info>", OutputInterface::VERBOSITY_VERBOSE);
```
Comment (ChristophWurst) [inline apps/files/lib/Command/ScanAppData.php]: same

File: apps/files/lib/Command/ScanAppData.php [hunk 3]
```diff
@@ -148,33 +127,19 @@ protected function scanFiles($verbose, OutputInterface $output) {
...
 		# restrict the verbosity level to VERBOSITY_VERBOSE
 		if ($output->getVerbosity() > OutputInterface::VERBOSITY_VERBOSE) {
 			$output->setVerbosity(OutputInterface::VERBOSITY_VERBOSE);
 		}
-		if ($quiet) {
-			$verbose = false;
-		}
 
-		$output->writeln("\nScanning AppData for files");
+		$output->writeln('Scanning AppData for files');
```
Comment (ChristophWurst) [inline apps/files/lib/Command/ScanAppData.php]: same

File: apps/files/lib/Command/ScanAppData.php [hunk 4]
```diff
@@ -213,7 +178,6 @@ public function exceptionErrorHandler($severity, $message, $file, $line) {
 	protected function presentStats(OutputInterface $output) {
 		// Stop the timer
 		$this->execTime += microtime(true);
-		$output->writeln("");
```
Comment (ChristophWurst) [inline apps/files/lib/Command/ScanAppData.php]: same

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Please don't mix such changes. Let's just adjust the verbosity output logic in this PR.

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Please don't change the command logic (post increment, output formatting) in this PR :wink:

**Comment by MorrisJobke** [comment]
Let's close this one in favor of #11831

**Comment by MorrisJobke** [comment]
> Let's close this one in favor of #11831

I just realized that it is for a different command.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍 Code looks good

**Comment by danielkesselberg** [comment]
@ChristophWurst :ping_pong:

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**